src/scripts/bttvIntegration.js

```diff
@@ -0,0 +1,105 @@
+function replaceKeywordWithEmoteImageString(textContent, keyword, imageSource, id){ 
```

Comment:
I love how you've broken up the logic in this file. Would you be open to adding JSDoc comments to each of your functions to add a bit more type support/intellisense?

Reply:
Absolutely! It'll be my first time with that but I think I can just reverse it based on the functions you've done that for, will submit and let me know if something's off

Reply:
https://jsdoc.app is a fantastic resource, too!

Reply:
Great job on these JSDocs! I've left some comments on some slight improvements that could be made, but overall, very well done!